app/helpers/discovered_hosts_helper.rb

```diff
@@ -44,4 +44,23 @@ def discovery_attribute(host, attr, default_value = _('N/A'))
     host.try(:discovery_attribute_set).try(attr) || default_value
   end
 
+  def discovery_status_icon(host)
+    if host.created_at > 1.day.ago
+      status_glyph = 'glyphicon-plus-sign'
+      status_message = _('New in the last 24 hours')
+      status_color = '#89A54E'
+    elsif host.last_report < 7.days.ago
+      status_glyph = 'glyphicon-exclamation-sign'
+      status_message = _('Not reported in more than 7 days')
+      status_color = '#AA4643'
+    else
+      status_glyph = 'glyphicon-ok-sign'
+      status_message = _('Reported in the last 7 days')
+      status_color = '#4572A7'
+    end
+
+    "<span class='glyphicon #{status_glyph}' style='color: #{status_color}'
```

Comment:
Was thinking like:

    "<span class='glyphicon #{status_glyph}' style='color: #{status_color};padding-left:10px '

But I realized this won't work. In that case use `&thinsp;` instead which looks much better.